examples/node_failure_shutdown.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
   annotations:
     chaos.datadoghq.com/environment: "lima"
 spec:
+  level: node # selector targets nodes instead of pods. Targeting a pod will impact the node hosting given pod.
```

Comment:
If you're changing the level, you need to change the selector to not be a pod selector anymore